src/test/java/org/folio/logintest/CredentialExistenceTest.java

```diff
@@ -56,21 +56,23 @@ public static void setup(final TestContext context) {
 
     try {
       PostgresClient.setIsEmbedded(true);
-      PostgresClient.getInstance(vertx).startEmbeddedPostgres();
+      PostgresClient.getInstance(vertx);
     } catch (Exception e) {
       context.fail(e);
     }
 
-    TenantClient tenantClient = new TenantClient(okapiUrl, TENANT, "token");
     DeploymentOptions restVerticleDeploymentOptions =
       new DeploymentOptions().setConfig(new JsonObject().put("http.port", port));
     vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, res -> {
       try {
         TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
-        tenantClient.postTenant(ta, handler -> {
+        TenantAPI tenantAPI = new TenantRefAPI();
+        Map<String, String> okapiHeaders = Map.of("x-okapi-url", okapiUrl,
+            "x-okapi-tenant", TENANT);
+        tenantAPI.postTenantSync(ta, okapiHeaders, handler -> {
           saveCredential(EXISTING_CREDENTIALS_USER_ID, "password")
             .onComplete(context.asyncAssertSuccess(done -> async.complete()));
-        });
+        }, vertx.getOrCreateContext());
       } catch (Exception e) {
```

Comment:
You can avoid the try catch if you like. postTenantSync throws nothing (except for system exceptions of course).

I'll just say this once here.. There are many places where you can avoid that try - catch. Not a big deal.